target/linux/ath79/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -4,93 +4,7 @@
 
```

Comment:
You could move this commit before the gl-ar750s one so that you don't ended up creating ath10k-caldata and immediately deleting a huge part of it.

Reply:
I'll have to wait and see what @adrianschmutzler's changes do. He's apparently refactoring the routines and I don't have much appetite for refactoring twice for what seems to be a purely cosmetic reason.